src/components/Dialogs/ConnectionStatusDialog.vue

```diff
@@ -25,9 +25,28 @@ const connectionStatusColor = computed(() => {
   }
 })
 
+const geoDetails = ref<string | null>(null)
+
 const close = () => {
   isOpened.value = false
 }
+
+onMounted(() => {
+  getGeoDetails()
+})
+
+const getGeoDetails = () => {
+  const externalIp = logStore.externalIp
```

Comment:
What if it wasn't able to extract an IP?

Reply:
Moved script to logStore to watch for a change in externalIP. Else, display "error fetching" message